package.json

```diff
@@ -84,6 +84,6 @@
         "@types/mocha": "^2.2.48",
         "@types/node": "^7.0.67",
         "typescript": "^2.9.2",
-        "vscode": "^1.1.18"
+        "vscode": "^1.1.33"
```

Comment:
Could you revert this too, please. I'll do a version update separately soon and check the dependencies.

Reply:
Actually nevermind that. I'll make the same change regardless. Could you commit the package-lock.json change alongside this after an `npm update`?

Reply:
Added package-lock.json.